src/main/java/teammates/common/datatransfer/questions/FeedbackRankOptionsQuestionDetails.java

```diff
@@ -75,16 +75,16 @@ public boolean extractQuestionDetails(Map<String, String[]> requestParameters,
         this.initialiseQuestionDetails(options);
 
         String minOptionsToBeRanked = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSTOBERANKED);
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_MIN_OPTIONS_TO_BE_RANKED);
         String maxOptionsToBeRanked = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKMAXOPTIONSTOBERANKED);
-        String isMinOptionsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKISMINOPTIONSTOBERANKEDENABLED);
-        String isMaxOptionsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKISMAXOPTIONSTOBERANKEDENABLED);
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_MAX_OPTIONS_TO_BE_RANKED);
+        String minOptionsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_IS_MIN_OPTIONS_TO_BE_RANKED_ENABLED);
+        String maxOptionsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_IS_MAX_OPTIONS_TO_BE_RANKED_ENABLED);
 
-        this.minOptionsToBeRankedEnabled = "on".equals(isMinOptionsToBeRankedEnabled);
-        this.maxOptionsToBeRankedEnabled = "on".equals(isMaxOptionsToBeRankedEnabled);
+        this.isMinOptionsToBeRankedEnabled = "on".equals(minOptionsToBeRankedEnabled);
```

Comment:
This gets shadowed here (and you can remove `this`, it's not required):
https://github.com/TEAMMATES/teammates/blob/2ed552ed0950836a726fc0a8f16ad71ef5ae174a/src/main/java/teammates/common/datatransfer/questions/FeedbackRankOptionsQuestionDetails.java#L128-L129

Reply:
We actually need to store `isMinOptionsToBeRankedEnabled ` as a class variable because we need to do backend validation.

Instead of relying on determining whether `minOptionsToBeRanked` has a value or not,
`isMinOptionsToBeRankedEnabled ` is used to check that if `isMinOptionsToBeRankedEnabled=True` but `minOptionsToBeRanked` has no value, then a warning statusMessage is returned.

In the case that you highlighted, it is appropriate to use it that way because its used to populate the form (and we do not store `isMinOptionsToBeRankedEnabled`) in the DB.

Reply:
Sorry I didn't catch this previously, what I meant was to remove `this`, not remove the instance variable so what you mentioned is fine.

However, the problem is that the instance variable is shadowed by the local variable.